scipy/optimize/_tnc.py

```diff
@@ -32,7 +32,7 @@
 (as a list of values); or None, to abort the minimization.
 """
 
-from scipy.optimize import moduleTNC
+from scipy.optimize import _moduleTNC as moduleTNC
```

Comment:
This is a straight revert; I considered dropping the `as moduleTNC`, but that would break the backwards compat import in `optimize/tnc.py`.